config/metac.yaml

```diff
@@ -11,6 +11,56 @@ spec:
   attachments:
   - apiVersion: dao.mayadata.io/v1alpha1
     resource: openebses
+  - apiVersion: apps/v1
+    resource: daemonsets
+    updateStrategy:
+      method: InPlace
+    advancedSelector:
+      selectorTerms:
```

Comment:
are these attachments installed in the watch's namespace? or some field in watch's specs? or based on some field in watch's status?
@sagarkrsd we can discuss more on this design

Reply:
Sure @amitd

Reply:
CSI Driver(Statefulset, Daemonset, ...) will be in other namespaces than watch's namespace.